launcher/game/tl/spanish/launcher.rpy

```diff
@@ -1730,3 +1730,14 @@ translate spanish strings:
     old "Automatically installing expansion APKs {a=https://issuetracker.google.com/issues/160942333}may not work on Android 11{/a}."
     new "Es posible que la instalación automática de APK de expansión {a=https://issuetracker.google.com/issues/160942333} no funcione en Android 11 {/a}."
 
+    # game/preferences.rpy:199
+    old "Default theme"
+    new "Tema predeterminado "
```

Comment:
Remove the space at the end of the string.

Reply:
Ok